ddpui/api/charts_api.py

```
@@ -785,6 +787,88 @@ def generate_map_chart_data(request, payload: ChartDataPayload):
+
+                # Fetch next page
+                page += 1
+                preview_data = charts_service.get_chart_data_table_preview(
+                    org_warehouse, payload, page=page, limit=page_size
+                )
+                data = preview_data["data"]
+
+            output.close()
+
+        except Exception as error:
+            logger.exception(f"Error streaming chart data: {error}")
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)